__testfixtures__/literal.output.js

```diff
@@ -0,0 +1,4 @@
+import Model, { attr, belongsTo, hasMany } from '@ember-data/model';
```

Comment:
I updated this output to show the failing test.  My bad, this isn't ready to go.  I ran it on our project and had to manually collapse so we didn't have duplicate imports.  I'll try to fix.

Reply:
Now worries, lmk when you need another review.